docs/app/Components/ComponentDoc/ComponentDoc.js

```diff
@@ -47,8 +47,16 @@ class ComponentDoc extends Component {
     ).isRequired,
     suiLink: PropTypes.string,
   }
-
-  state = {}
+  constructor(props) {
+    super(props)
+    let activePath
+    if (location.hash) {
+      activePath = _.last((location.hash || '').split('#'))
+    }
+    this.state = {
+      activePath,
+    }
+  }
```

Comment:
Prefer property initializer for state unless `props` are needed to calculate state:

```jsx
state = {
  activePath: _.last((location.hash || '').split('#'))
}
```